controller/src/main/java/io/pravega/controller/store/client/StoreClientFactory.java

```diff
@@ -62,6 +70,20 @@ private static CuratorFramework createZKClient(ZKClientConfig zkClientConfig) {
         return zkClient;
     }
 
+    private static String loadPasswdFromFile(String trustStorePasswordPath) {
+        byte[] pwd;
```

Comment:
you use `pwd`, `passwd` and `password` all within 2 lines. Please be consistent. Let's use the full word. This includes the name of this method.